site.js

```diff
@@ -135,6 +135,11 @@ $(document).ready(function (){
         {content: '3.1', start: '2019-01-01', end: '2019-07-01'},
         {content: '3.2', start: '2019-07-01', end: '2020-01-01'}
     ],
+    'Spyder':[
+        {content: '3', start: '2016-09-24', end: '2019-03-01', py2:true},
```

Comment:
I think the end date of our 3 series is going to be 2018-12-01.

Reply:
The problem is that if we define the "support" date as the day of the last scheduled release, will leave a gap between them, rather than being supported at least until the release of the following version (i.e. leaving no version of Spyder "supported" during the transition period), unlike the convention followed by all the other projects on this list. Even if the last scheduled release is on that date, we would naturally support Spyder 3 (e.g. via Github, Gitter, Google Groups, Docs, and possibly emergency fixes if necessary) *at least* until the release of Spyder 4.

Therefore, I propose an end date of at least 2019-01-15, the same day on which Spyder 4 is scheduled for release (although at least in practice, we'd want to consider the delay to get ``conda`` packages out, possible emergency fixes, at least some latency in all users upgrading to a new major version on the day of its release, etc.).